pipelines/generate_test_data.R

```diff
@@ -70,7 +70,7 @@ create_facility_test_data <- function(
   target_fac_data <- tibble(
     reference_date = reference_dates,
     report_date = end_reference,
-    geo_type = "state",
+    geo_type = "loc",
```

Comment:
This should still be state since we're mirroring gold tables, no?
```suggestion
    geo_type = "state",
```